python/servo/command_base.py

```diff
@@ -618,48 +605,31 @@ def build_android_env_if_needed(self, env: Dict[str, str]):
             host_suffix = "x86_64"
         host = os_type + "-" + host_suffix
 
-        host_cc = env.get('HOST_CC') or shutil.which(["clang"]) or util.whichget_exec_path(["gcc"])
-        host_cxx = env.get('HOST_CXX') or util.whichget_exec_path(["clang++"]) or util.whichget_exec_path(["g++"])
-
-        llvm_toolchain = path.join(env['ANDROID_NDK'], "toolchains", "llvm", "prebuilt", host)
-        gcc_toolchain = path.join(env['ANDROID_NDK'], "toolchains",
-                                  android_toolchain_prefix + "-4.9", "prebuilt", host)
-        gcc_libs = path.join(gcc_toolchain, "lib", "gcc", android_toolchain_name, "4.9.x")
+        host_cc = env.get('HOST_CC') or shutil.which("clang") or shutil.which(["gcc"])
+        host_cxx = env.get('HOST_CXX') or shutil.which("clang++") or shutil.which("g++")
```

Comment:
I think we should remove the `gcc` and `g++` fallbacks here, as I don't know if Servo supports gcc any longer.